src/hal/src/sensors_cf2.c

```diff
@@ -398,7 +405,7 @@ static void sensorsSetupSlaveRead(void)
   // SMPLRT_DIV is only used for 1kHz internal sampling." Slowest update rate is then 500Hz.
   mpu6500SetSlave4MasterDelay(15); // read slaves at 500Hz = (8000Hz / (1 + 15))
 #else
-  mpu6500SetSlave4MasterDelay(4); // read slaves at 100Hz = (500Hz / (1 + 4))
+  mpu6500SetSlave4MasterDelay(9); // read slaves at 100Hz = (500Hz / (1 + 4))
```

Comment:
Please update comment as well. Faster sampling rate might improve latency? Reason why you set it to 100Hz?